source/agora/utils/WellKnownKeys.d

```diff
@@ -1434,6 +1434,29 @@ static immutable Genesis = KeyPair(
     Seed([167, 197, 41, 45, 194, 231, 7, 114, 117, 27, 234, 152, 100, 142, 108, 235, 91, 123, 216, 92, 108, 4, 54, 200, 217, 114, 60, 126, 156, 76, 184, 148]));
 
 
+/*******************************************************************************
+
+    Commons Budget KeyPair used in unittests
+
+    In unittests, we need the commons budget key pair to be known for us to be
+    able to write tests.
+    In the real network, there are different values.
+
+    Note that while this is a well-known keys, it is not part of the
+    range returned by `byRange`, nor can it be indexed by `size_t`,
+    to avoid it being mistakenly used.
+
+    Seed:    SCNRULE3Q7NBNX7UIJVVI6HI5DQKP3MNDEHJA66FJZXDEGGH4SRRDSDR
+    Address: GCOMMONBGUXXP4RFCYGEF74JDJVPUW2GUENGTKKJECDNO6AGO32CUWGU
+
+*******************************************************************************/
+
+static immutable CommonsBudgetKeyPair = KeyPair(
```

Comment:
Why this name ? **ALL** other KeyPair are named in a human-readable way. `NODE{2..7}`, `A`..`Z`, `AA`..`ZZ`... None of them has a `KeyPair` suffix, so why does this one have one ?

Reply:
> Why this name ? **ALL** other KeyPair are named in a human-readable way. `NODE{2..7}`, `A`..`Z`, `AA`..`ZZ`... None of them has a `KeyPair` suffix, so why does this one have one ?

I see. I'll change it to CommonsBudget.